### ISSUE TITLE
investigate unusual header on Bench.scala

### ISSUE THREAD

**Issue by pjfanning** [apache/incubator-pekko#157]
https://github.com/apache/incubator-pekko/blob/main/actor-tests/src/test/scala/org/apache/pekko/actor/Bench.scala

```
/* The Computer Language Benchmarks Game
   http://shootout.alioth.debian.org/
   contributed by Julien Gaugaz
   inspired by the version contributed by Yura Taras and modified by Isaac Gouy
 */
```

Doesn't appear to assert a copyright or a license.

We can keep it but maybe don't need to.

If it is decided that this does assert a copyright and/or a license, we may need to update the LICENSE or NOTICE for this repo.

If this is not a valid copyright and/or license - then we should remove the check in project/CopyrightHeader.scala

**Comment by justinmclean** [comment]
Hi,

You do need to know how it is licensed. Looks to be this? Although the names are different.

https://salsa.debian.org/benchmarksgame-team/archive-alioth-benchmarksgame/-/blob/master/LICENSE.md <https://salsa.debian.org/benchmarksgame-team/archive-alioth-benchmarksgame/-/blob/master/LICENSE.md>

Justin

**Comment by jrudolph** [comment]
If in doubt, it would probably also ok to just remove it, it's just benchmark code.

**Comment by pjfanning** [comment]
it is basically https://salsa.debian.org/benchmarksgame-team/archive-alioth-benchmarksgame/-/blob/master/contributed-source-code/shootout/chameneos/chameneos.scala-2.scala

I suspect removing this benchmark test might be the best way forward.

**Comment by justinmclean** [comment]
The license is compatible with the Apache license you could leave it in and add that information to LICENSE

**Comment by pjfanning** [comment]
I'm running a build, it so far looks like that file is not even used.

**Comment by jrudolph** [comment]
Fixed by removing the file in #171.
